### PR TITLE
Added version increment to Travis CI release flow based on latest tag

### PR DESCRIPTION
**Pull request by adrian-shyshkovskyi-here**
Signed-off-by: Adrian Shyshkovskyi <ext-adrian.shyshkovskyi@here.com>

### CODE REVIEW

File: .travis/scripts/prepare_release.sh [hunk 1]
```diff
@@ -2,6 +2,20 @@
 set -ev
 
 # Prepare release
+LATEST_RELEASE_TAG=$(git describe --abbrev=0)
```
Comment (dmitriy-abramov) [inline .travis/scripts/prepare_release.sh]: I would better call the variable PREVIOUS_RELEASE_TAG

File: .travis/scripts/prepare_release.sh [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+  # Increment patch version
+  mvn -B -q build-helper:parse-version versions:set \
+  -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion} \
+  versions:commit
+else
+  echo "Cannot parse the latest release tag: ${LATEST_RELEASE_TAG}"
+  exit 1
+fi
+
 RELEASE_VERSION=$(mvn -q \
```
Comment (dmitriy-abramov) [inline .travis/scripts/prepare_release.sh]: RELEASE_TAG

File: .travis/scripts/prepare_release.sh (hunk 1 above)
Comment (adrian-shyshkovskyi-here) [reply .travis/scripts/prepare_release.sh]: Done

File: .travis/scripts/prepare_release.sh (hunk 2 above)
Comment (adrian-shyshkovskyi-here) [reply .travis/scripts/prepare_release.sh]: Done

**Pull request merged**